pkg/cmd/taskrun/describe_test.go

```diff
@@ -731,3 +731,99 @@ func TestTaskRunWithSpecDescribe_custom_timeout(t *testing.T) {
 	}
 	golden.Assert(t, actual, fmt.Sprintf("%s.golden", t.Name()))
 }
+
+func TestTaskRunDescribe_last(t *testing.T) {
+	clock := clockwork.NewFakeClock()
+
+	/*
```

Comment:
This comment isn't necessary. This test data will be sorted by the command run in this test, so it will be ordered by start time regardless of how elements are added to the TaskRun list.